pkg/oc/cli/admin/release/new.go

```diff
@@ -101,8 +111,9 @@ func NewRelease(f kcmdutil.Factory, parentName string, streams genericclioptions
 	flags := cmd.Flags()
 
 	// image inputs
-	flags.StringSliceVarP(&o.Filenames, "filename", "f", o.Filenames, "A file defining a mapping of input images to use to build the release")
+	flags.StringSliceVar(&o.Filenames, "filename", o.Filenames, "A file defining a mapping of input images to use to build the release")
 	flags.StringVar(&o.FromImageStream, "from-image-stream", o.FromImageStream, "Look at all tags in the provided image stream and build a release payload from them.")
+	flags.StringVarP(&o.FromImageStreamFile, "from-image-stream-file", "f", o.FromImageStreamFile, "Take the provided image stream on disk and build a release payload from it.")
```

Comment:
This will be confusing with all the commands which use `-f` for `--filename`. I'd strongly recommend we either don't have a shorthand or have a different one. Also how `--from-image-stream-file` differs from `--from-image-stream` is it only for the fact that one points to IS on disk and another to on server, what's the story behind that?

Reply:
What happens when I specify both?

Reply:
We don't have any arguments that take both a file and a name on the server.  `-f` is the filename shorthand - you're creating a release from a file.  I suppose I could rename --filename to be something else, but this would still probably need to indicate it's a stream.

Reply:
Renamed `--filenames` to be `--mapping-file` but I want to leave `--from-image-stream-file` to be consistent with the other sources.  `-f` is the right shortcut for that though.